### PR TITLE
Add search text in source feeds

### PR DESCRIPTION
**Pull request by barshathakuri**
## Addresses:
- #82 
- #80 

## Changes
- Add search text in source feeds

## This PR doesn't introduce:
- [x] typos
- [x] conflict markers
- [x] unwanted comments
- [x] temporary files, auto-generated files or secret keys
- [x] `console.log` meant for debugging
- [x] codegen errors

### CODE REVIEW

File: src/views/AllSourcesFeeds/SourceCard/index.tsx [hunk 1]
```diff
@@ -27,6 +27,7 @@ function SourceCard(props: Props) {
         <Link
             className={styles.sourceCard}
             to={data?.url}
+            target="_blank"
```
Comment (samshara) [inline src/views/AllSourcesFeeds/SourceCard/index.tsx:30]: Let's add `external` prop and remove this target prop.

**Review by samshara — DISMISSED** [review]

File: src/views/Home/AlertsTable/index.tsx [hunk 2]
```diff
@@ -263,7 +281,7 @@ function AlertsTable() {
         >
             <SortContext.Provider value={sortState}>
                 <Table
-                    pending={loading}
+                    pending={alerInfoLoading}
```
Comment (puranban) [inline src/views/Home/AlertsTable/index.tsx]:
```suggestion
                    pending={alertInfoLoading}
```

**Review by puranban — CHANGES_REQUESTED** [review]

File: src/views/Home/AlertsTable/index.tsx [hunk 3]
```diff
@@ -154,9 +158,10 @@ function AlertsTable() {
     ]);
 
     const {
-        loading,
+        loading: alerInfoLoading,
```
Comment (puranban) [inline src/views/Home/AlertsTable/index.tsx]:
```suggestion
        loading: alertInfoLoading,
```

**Review by puranban — CHANGES_REQUESTED** [review]

File: src/views/AllSourcesFeeds/SourceCard/index.tsx (hunk 1 above)
Comment (puranban) [reply src/views/AllSourcesFeeds/SourceCard/index.tsx:30]: resolving for now because the link is actual internal

**Review by puranban — APPROVED** [review]
looks good

**Pull request merged**